### PR TITLE
Updating OVA deployment doc to reflect current state.

### PR DESCRIPTION
**Pull request by stuclem**
Updated the OVA docs to include the new sections for the interactive VCH installer and KoV. Included that VC credentials are required at the end of the installation process. Some content reorg.

- Fixes https://github.com/vmware/vic-product/issues/396
- Towards https://github.com/vmware/vic-product/issues/286, pending the completion of the Getting Started page.

### CODE REVIEW

**Comment by stuclem** [comment]
@andrewtchin can you please take a look?

File: docs/user_doc/vic_vsphere_admin/deploy_demo_vch.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Deploy a Demo Virtual Container Host
+# Deploy Virtual Container Hosts Interactively
 
 The vSphere Integrated Containers appliance provides an interactive web installer from which you can deploy a basic virtual container host (VCH). This VCH has limited cabilities and is for demonstration purposes only, to allow you to start experimenting with vSphere Integrated Containers.
```
Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_demo_vch.md]: I would add a note that VCHs deployed with this should not be used in production

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 2]
```diff
@@ -18,53 +21,69 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
...
 6. Expand **Registry Configuration** to configure the deployment of vSphere Integrated Containers Registry. 
 
-    - If you do not want to deploy vSphere Integrated Containers Registry, uncheck the **Deploy Registry** check box.
+    - If you do not want to enable vSphere Integrated Containers Registry, uncheck the **Deploy Registry** check box.
     - In the **Registry Port** text box, optionally change the port on which to publish the vSphere Integrated Containers Registry service.
     - In the **Notary Port** text box, optionally change the port on which to publish the Docker Notary service for vSphere Integrated Containers Registry.
-    - In the **Registry Admin Password** text box, set the password for the vSphere Integrated Containers Registry admin account.
-    - In the **Database Password** text box, set the password for the root user of the MySQL database that vSphere Integrated Containers Registry uses.
-    - Optionally check the **Garbage Collection** check box to enable garbage collection when the appliance reboots. 
+    - In the **Registry Admin Password** text box, set a password for the vSphere Integrated Containers Registry admin account.
```
Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: This field is removed

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 3]
```diff
@@ -18,53 +21,69 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
...
+    - In the **Database Password** text box, set a password for the root user of the MySQL database that vSphere Integrated Containers Registry uses.
```
Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: `Database Password` field is removed as we autogenerate it and store it in the config

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 4]
```diff
@@ -18,53 +21,69 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
...
+    - Optionally check the **Garbage Collection** check box to enable garbage collection on the registry when the appliance reboots.
```
Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Is there a note somewhere else that if the use wants to later enable GC they can go to Edit Settings for the appliance and check it then when they restart Harbor GC will happen?

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 5]
```diff
@@ -18,53 +21,69 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
...
+    - In the **Registry Admin Password** text box, set a password for the vSphere Integrated Containers Registry admin account.
+    - In the **Database Password** text box, set a password for the root user of the MySQL database that vSphere Integrated Containers Registry uses.
+    - Optionally check the **Garbage Collection** check box to enable garbage collection on the registry when the appliance reboots. 
     - To use custom certificates to authenticate connections to vSphere Integrated Containers Registry, optionally paste the content of the appropriate certificate and key files in the **SSL Cert** and **SSL Cert Key** text boxes. Leave the text boxes blank to use auto-generated certificates. 
 
 7. Expand **Management Portal Configuration** to configure the deployment of vSphere Integrated Containers Management Portal. 
 
-    - If you do not want to deploy vSphere Integrated Containers Management Portal, uncheck the **Deploy Management Portal** check box.
+    - If you do not want to enable vSphere Integrated Containers Management Portal, uncheck the **Deploy Management Portal** check box.
```
Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Deploying Admiral will now be required in 1.2 and Harbor may also be required, I'll check today

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 6]
```diff
@@ -18,53 +21,69 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
...
     - In the **Management Portal Port** text box, optionally change the port on which to publish the vSphere Integrated Containers Management Portal service.
     - To use custom certificates to authenticate connections to vSphere Integrated Containers Management Portal, optionally paste the content of the appropriate certificate and key files in the **SSL Cert** and **SSL Cert Key** text boxes. Leave the text boxes blank to use auto-generated certificates.
-7. Expand **Fileserver Configuration** to configure the file server from which you download vSphere Integrated Containers Engine and which publishes the plug-in packages for the vSphere Client. 
+7. Expand **Fileserver Configuration** to configure the file server from which you download the vSphere Integrated Containers Engine and vSphere Container Cluster binaries, and which publishes the plug-in packages for the vSphere Client.
```
Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: I'm not sure how we want to document this, but the main fileserver page at `:9443` is now the "Getting Started Page", while the directory listing view is moved to `:9443/files`

Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Oh haha I see below

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 7]
```diff
@@ -18,53 +21,69 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
...
+8. Expand **vSphere Container Cluster Manager Configuration** to configure the deployment of vSphere Container Cluster Manager.
+
+    - If you do not want to enable vSphere Container Cluster Manager, uncheck the **Deploy vSphere Container Cluster Manager** check box.
+    - In the **Admin username** text box, optionally specify a user name for an account for vSphere Container Cluster Manager to use when accessing the Kubernetes API . 
+    
+      If not specified, the installer creates an account named `admin`.
+    - To use custom certificates to authenticate connections to vSphere Container Cluster Manager, optionally paste the content of the appropriate certificate and key files in the **SSL Cert** and **SSL Cert Key** text boxes. Leave the text boxes blank to use auto-generated certificates.
 8. Click **Next** and **Finish** to deploy the vSphere Integrated Containers appliance.
 9. When the deployment completes, power on the appliance VM.
+10. Go to https://<i>vic_appliance_address</i>:9443 and enter the address and single sign-on credentials of the vCenter Server instance on which you deployed the appliance.
```
Comment (andrewtchin) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Also maybe note that Harbor and Admiral will not start and the web interfaces will not be available until after this step is completed

File: docs/user_doc/vic_vsphere_admin/deploy_demo_vch.md (hunk 1 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_demo_vch.md]: We include the following a couple of lines further down:

**IMPORTANT**: The demo VCH does not implement any TLS authentication options, and as such is completely unsecured. Do not use the demo VCH in production environments. To deploy VCHs for use in production environments, use the `vic-machine` command line utility.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md (hunk 3 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: It's still there in vic-746-7f90d4b1.ova, from 21st July. I see there's a newer build, so am downloading that one now.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md (hunk 5 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: The checkboxes are still there in vic-746-7f90d4b1.ova, so I'll leave them here for the time being.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md (hunk 4 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Documented in https://vmware.github.io/vic-product/assets/files/html/1.2/vic_vsphere_admin/reconfigure_appliance.html, but I added a line at the very end of the topic to say that it's possible (and not just for GC).

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md (hunk 6 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Yes, these updates will be addressed in https://github.com/vmware/vic-product/issues/286, once the Getting Started page is complete.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md (hunk 7 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Done

**Pull request merged**